salt/modules/dockerng.py

```diff
@@ -504,10 +504,14 @@ def __virtual__():
                 log.warning(
                     'Insufficient Docker version for dockerng (required: '
                     '{0}, installed: {1})'.format(
-                        '.'.join(docker_versioninfo),
-                        '.'.join(MIN_DOCKER)
-                    )
-                )
+                        '.'.join(map(str, MIN_DOCKER)),
+                        '.'.join(map(str, docker_versioninfo))))
+        else:
+            log.warning(
```

Comment:
I am not sure this is documented yet, but it is a feature that was added along with some more significant loader changes.  The idea being to prevent unneeded/confusing logging messages while still supplying a user who needs a module a reason why it doesn't load.